### ISSUE TITLE
[6.2.z][6.1.z] fix nonexisting rhel65 image

### ISSUE THREAD

**Issue by lpramuk** [SatelliteQE/robottelo#3899]
We have a lot of ```robottelo/vm.py:147: VirtualMachineError```s in 6.2 automation caused by  attempts to provision VM based on nonexisting ```rhel65``` image:
```
 VirtualMachineError: Failed to fetch virtual machine IP address information
```

**Comment by kbidarkar** [comment]
We have fixes for this issue, these need to be cherry-picked.

**Comment by lpramuk** [comment]
OK, then I'm going to cherrypick PR #3856 to 6.2.z and 6.1.z branches

**Comment by lpramuk** [comment]
I'm done, cherrypicked and lot of conflicts resolved.

**Comment by lpramuk** [comment]
Both PRs backporting fix to 6.1.z and 6.2.z branch are merged now.
Closing.

**Comment by omaciel** [comment]
@lpramuk shouldn't you be sleeping now? :)
